crates/sncast/tests/e2e/account/helpers.rs

```diff
@@ -0,0 +1,67 @@
+use indoc::indoc;
+use tempfile::{tempdir, TempDir};
+use tokio::fs::File;
+use tokio::io::AsyncWriteExt;
+
+#[must_use]
+pub async fn create_tempdir_with_empty_json(file_name: &str) -> TempDir {
+    let tempdir = tempdir().expect("Unable to create temporary directory");
+
+    let data = r"{}";
+
+    let mut file = File::create(tempdir.path().join(file_name))
+        .await
+        .expect("Could not create temporary accounts file!");
+
+    file.write_all(data.as_bytes())
+        .await
+        .expect("Could not write temporary testing accounts");
+
+    let _ = file.flush().await;
```

Comment:
You do not check whether it is an `Err` or `Ok`.

Reply:
I prefer to add a short message just in case the call doesn't succeed somehow, despite the rarity of such scenario

Reply:
That's what I mean, but currently, the call can fail, and the function will not panic. So add `unwrap` or `expect`